modules/exploits/windows/http/desktopcentral_deserialization.rb

```diff
@@ -83,26 +82,29 @@ def check
     )
 
     unless res
-      return CheckCode::Unknown('Target is not responding to check')
+      return CheckCode::Unknown('Target did not respond to check request.')
     end
 
     unless res.code == 200 && res.body.include?('ManageEngine Desktop Central')
-      return CheckCode::Unknown('Target is not running Desktop Central')
+      return CheckCode::Unknown('Target is not running Desktop Central.')
     end
 
-    version = res.get_html_document.at('//input[@id = "buildNum"]/@value')&.text
+    build = res.get_html_document.at('//input[@id = "buildNum"]/@value')&.text
 
-    unless version
-      return CheckCode::Detected('Could not detect Desktop Central version')
+    unless (build = Integer(build) rescue nil)
+      return CheckCode::Detected(
+        'Target did not respond with Desktop Central build.'
+      )
     end
 
-    vprint_status("Detected Desktop Central version #{version}")
-
-    if Gem::Version.new(version) < notes['PatchedVersion']
-      return CheckCode::Appears("#{version} is an exploitable version")
+    # Desktop Central build 100474 is equivalent to version 10.0.474
```

Comment:
Extra comments appreciated, thanks for adding this in and setting the build check to be a bit more readable in the code.